.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "python.testing.pytestArgs": [
+        "ckan/tests"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true
+}
```

Comment:
```suggestion
}

```

Reply:
Fixed in 4490493cac43fe134f839c44b25291c96a687d51